webpack.config.js

```diff
@@ -37,6 +38,9 @@ const webpackConfig = {
 		chunkFilename: '[name].[chunkhash].js', // ditto
 		devtoolModuleFilenameTemplate: 'app:///[resource-path]'
 	},
+	postcss() {
```

Comment:
FYI this is invalid in more recent versions of Webpack 2.x which implement strict config validation. Instead you'll probably want to create a `.postcssrc` file.